stackdriver-core/test/google/cloud/configuration_test.rb

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
```

Comment:
The test files don't need the extra new line, since they aren't pulled into the documentation.